x-pack/plugins/enterprise_search/public/applications/__mocks__/kea.mock.ts

```diff
@@ -22,6 +23,7 @@ export const mockAllValues = {
   ...mockFlashMessagesValues,
 };
 export const mockAllActions = {
+  ...mockTelemetryActions,
```

Comment:
Why do you add these here if you just end up importing `telementry_logic.mock` directly in all of the tests?

Reply:
Because that way you import a single `__mocks__/kea.mock.ts` to mock all possible shared logic file values/actions at once instead of having to manually and tediously import every logic file that you need.

So for example, in product_card.test.tsx:

https://github.com/elastic/kibana/blob/1db25d72bdfbc9782ee76675a5256593160acdc4/x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_card/product_card.test.tsx#L7-L8

This file calls both `KibanaLogic` and `TelemetryLogic`. Because we have this shared `kea.mock.ts` helper, we only have to import it once and now the test file gracefully handles all instances of useValues and useActions, returning sane defaults that we can spy on via our exported `mock*Values`/`mock*Actions` objs.